server/src/main/kotlin/tech/figure/objectstore/gateway/sql/Varchar.kt

```diff
@@ -0,0 +1,17 @@
+package tech.figure.objectstore.gateway.sql
+
+import org.jetbrains.exposed.dao.Entity
+import org.jetbrains.exposed.dao.EntityClass
+import org.jetbrains.exposed.dao.id.EntityID
+import org.jetbrains.exposed.dao.id.IdTable
+import org.jetbrains.exposed.sql.Column
+
+open class VarcharTable(name: String, columnName: String, columnLength: Int) : IdTable<String>(name) {
```

Comment:
sometimes I wonder why things like this don't just already exist in Exposed by default...

Reply:
Right?  This one especially surprised me, because I've done things like this so many times in the past when it was like a table of configurations or something